NitroxServer/GameLogic/StoryTimingData.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxModel.Logger;
+using ProtoBufNet;
+
+namespace NitroxServer.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class StoryTimingData
+    {
+        [ProtoMember(1)]
+        public double ElapsedTime { get; set; }
+
+        [ProtoMember(2)]
+        public double? AuroraExplosionTime { get; set; }
```

Comment:
Do we need to store this?  Can it be inferred by the ElapsedTime?

Reply:
No, because of the random explode time, even if I were to use a "time remaining", it still affects the multiplication stuff. We can hardcode explode time but this is more accurate with what sn does.